src/main/MQ2ImGuiConsole.cpp

```diff
@@ -1328,6 +1348,49 @@ static void MakeColorGradient(float frequency1, float frequency2, float frequenc
 }
 
 //----------------------------------------------------------------------------
+ImGuiWindowFlags ConsoleFlagSetup()
+{
+	ImGuiWindowFlags windowFlags = 0;
+	if (!s_noTitleBar || s_mouseHoveringConsoleTimer != 0) windowFlags |= ImGuiWindowFlags_MenuBar;
+	if (s_noTitleBar)  windowFlags |= ImGuiWindowFlags_NoTitleBar;
+	if (s_lockConsole) windowFlags |= ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoResize;
+	return windowFlags;
+}
+
+void AlphaSetting(bool AlphaBool)
+{
+	if (AlphaBool || !s_alphaBool || s_mouseHoveringConsoleTimer != 0)
+	{
+		ImGui::SetNextWindowBgAlpha(1.0f);
+	}
+	else 
+	{
+		ImGui::SetNextWindowBgAlpha(.1f);
+	}
+}
+
+bool MouseOverUpdate()
```

Comment:
This should be a member funciton too, and have a name that reflects what it does more accurately. suggestion: UpdateMouseHoverConsoleTimer.